src/styles/_header.scss

```
@@ -37,5 +37,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: color 300ms;
```

та сама історія з color. Тут достатньо написати   transition-duration: 300ms;
```suggestion
      transition-duration: 300ms;
```

---

@YuriiPronin Always add transition property. By default, transition property is all, and it's hard for the browser to control all properties